tests/Microsoft.Fx.Portability.Offline.Tests/OffLineApiPortServiceTests.cs

```diff
@@ -29,23 +28,25 @@ public class OffLineApiPortServiceTests
             CreateApiRecommendations()
         );
 
-        [TestMethod]
-        [ExpectedException(typeof(ArgumentNullException))]
+        [Theory]
```

Comment:
This should be `[Fact]` because `[Theory]` expects some parameterized input. As a result, the tests fail for me, as is but work when I change them to Fact.